src/backend/distributed/executor/multi_utility.c

```diff
@@ -608,17 +611,18 @@ ProcessCopyStmt(CopyStmt *copyStatement, char *completionTag, bool *commandMustR
 
 
 /*
- * ProcessIndexStmt processes create index statements for distributed tables.
+ * PlanIndexStmt processes create index statements for distributed tables.
```

Comment:
Shall we also update the comments with the names given that we don't "execute" anything here? Similarly for other Plan_XXX_Stmt functions.